release-notes/topics/platform-intro.adoc

```diff
@@ -10,6 +10,6 @@ Red Hat Ansible Automation Platform simplifies the development and operation of
 |===
 | Ansible Automation Platform | Automation controller | Automation hub | Automation services catalog | Insights for Ansible Automation Platform
 
-|2.1.1 | 4.1.1 | 4.4.1 | hosted service | hosted service
+|2.2 | 4.2 | 4.4.2 | hosted service | hosted service
```

Comment:
ditto. Version of Hub is 4.5.0 for AAP 2.2.

Reply:
Fixed.